Scripts/Proxies/Vive/ViveProxy.cs

```diff
@@ -39,18 +39,9 @@ public override void Awake()
             var proxyHelper = m_LeftHand.GetComponent<ViveProxyHelper>();
             if (proxyHelper)
             {
-                foreach (var tooltip in proxyHelper.leftTooltips)
+                foreach (var @override in proxyHelper.leftPlacementOverrides)
```

Comment:
Not that it matters much, but why not just use a keyword that isn't reserved, like "placementOverride"?  Are @ literals acceptable in the conventions (or is there no verdict on their usage)? @AndrewTHEManeri thoughts?  (Not that it matters at the compiler level)

Reply:
Yeah, I've started using these mores just because it's all the same with autocomplete. placementOverride is fine here if we want to avoid @s

Reply:
I'd avoid the @s - there is a whole world of variable names we can use out there